plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -314,6 +315,30 @@ public static List<ColumnMetadata> getColumnMetadatas(Schema schema, TypeManager
         return columns.build();
     }
 
+    public static Schema updateColumnComment(Schema schema, String columnName, String comment)
+    {
+        NestedField fieldToUpdate = schema.findField(columnName);
+        checkArgument(fieldToUpdate != null, "Field %s does not exist", columnName);
+        NestedField updatedField =
+                NestedField.of(
+                        fieldToUpdate.fieldId(),
+                        fieldToUpdate.isOptional(),
+                        fieldToUpdate.name(),
+                        fieldToUpdate.type(),
+                        comment);
+
+        ImmutableList.Builder<NestedField> newFields = ImmutableList.builder();
+        newFields.add(updatedField);
+
+        for (NestedField field : schema.columns()) {
```

Comment:
This changes the order of the fields. Is that ok? It might be clearer to write this as
```java
List<NestedField> newFields = schema.columns().stream()
        .map(field -> (field.fieldId() == updatedField.fieldId()) ? updatedField : field)
        .toList();